### PR TITLE
HDF to csv converter. Change tool dependencies to be resolved by conda

### PR DESCRIPTION
**Pull request by maximskorik**
@martenson, could you take a look please. Does one have to specify conda collection when setting up dependencies (e.g. "conda-forge") in xml, or it is fine the way it is?

followup to https://github.com/RECETOX/galaxytools/pull/67

### CODE REVIEW

**Comment by maximskorik** [comment]
I accidentally pushed irrelevant files from my local directory. Sorry for the mess.

File: tools/hdf_converter/hdf_converter.xml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <tool id="apCLMS_to_RamClustR_converter" name="apLCMS to RamClustR converter" version="0.0.1" python_template_version="3.5">
     <description>converts aplcms HDF output to RamClustR csv input</description>
```
Comment (martenson) [inline tools/hdf_converter/hdf_converter.xml:2]: Is this true every time or is this applicable only to our wrappers of said tools? Maybe we could link them in `help` section to clarify this.

File: tools/hdf_converter/hdf_converter.xml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <tool id="apCLMS_to_RamClustR_converter" name="apLCMS to RamClustR converter" version="0.0.1" python_template_version="3.5">
```
Comment (martenson) [inline tools/hdf_converter/hdf_converter.xml:1]: the `id` here should match the one in .shed.yml

File: tools/hdf_converter/hdf_converter.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
     <description>converts aplcms HDF output to RamClustR csv input</description>
     <requirements>
-        <container type="docker">maxskorik/hdf_converter</container>
+        <requirement type="package" version="1.1.5">pandas</requirement>
+        <requirement type="package" version="3.6.1">pytables</requirement>
```
Comment (martenson) [inline tools/hdf_converter/hdf_converter.xml:5]: I'd explicitly declare Python here, you probably get one with pandas/pytables but this way you have no control over version.

Comment (hechth) [reply tools/hdf_converter/hdf_converter.xml:5]: How do you specifically control that is is a python package?

Comment (martenson) [reply tools/hdf_converter/hdf_converter.xml:5]: @hechth I don't understand the question, what I am suggesting is to include something like: `<requirement type="package" version="3.7">python</requirement>`

File: tools/hdf_converter/hdf_converter.xml [hunk 4]
```diff
@@ -24,6 +26,12 @@
         </test>
     </tests>
     <help><![CDATA[
-        Tool to convert apLCMS output to csv file formated in accordance to RamClustR input requirements.
+        Tool to convert apLCMS output to csv file formated in accordance to `RamClustR (CSV) <https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/ramclustr_csv/ramclustr_csv/1.1.0+galaxy0>`_ input requirements.
+
+        **Workflow position**
+
+        - Upstream tool: `apLCMS-unsupervised <https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_unsupervised/recetox_aplcms_unsupervised/6.6.6+galaxy1>`_ or `apLCMS-Hybrid <https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_hybrid/recetox_aplcms_hybrid/6.6.6+galaxy1>`_
```
Comment (martenson) [inline tools/hdf_converter/hdf_converter.xml:33]: These links are going to change as early as we switch to using the Main Tool Shed (and in many other circumstances). I recommend using links to github development repository as the canonical reference.

Comment (martenson) [reply tools/hdf_converter/hdf_converter.xml:33]: so e.g. https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr

**Comment by martenson** [comment]
> Does one have to specify conda collection when setting up dependencies (e.g. "conda-forge") in xml, or it is fine the way it is?

Galaxy uses [this](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/galaxy.yml.sample#L321) order of conda channels by default. So if the package is in `conda-forge` it will be installed from there. The order is by design.

**Comment by maximskorik** [comment]
> > Does one have to specify conda collection when setting up dependencies (e.g. "conda-forge") in xml, or it is fine the way it is?
> 
> Galaxy uses [this](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/galaxy.yml.sample#L321) order of conda channels by default. So if the package is in `conda-forge` it will be installed from there. The order is by design.

Alright, thanks for info

**Review by hechth — APPROVED** [review]

**Review by martenson — APPROVED** [review]
I took the liberty to squash this into a single commit, once the tests pass we can merge.

**Pull request merged**